docker-compose.yml

```diff
@@ -23,6 +23,8 @@ services:
         environment: 
```

Comment:
I believe you can use [`env_file`](https://docs.docker.com/compose/compose-file/#env_file) to run-time inject a `.env` file. I haven't used this myself so do a bit of research to see if this is the best method.  These seem to have some decent tutorials: https://medium.com/better-programming/using-variables-in-docker-compose-265a604c2006 and https://www.techrepublic.com/article/how-to-use-docker-env-file/ 

Another method we use at work (which probably isn't the best but works because of our constraints) is that we have a small `entrypoint.sh` script that runs as the entrypoint.

That script will run something like one of these (https://gist.github.com/mihow/9c7f559807069a03e302605691f85572): 
```bash
set -o allexport; source .env; set +o allexport
```
or similar and then the actual entrypoint.

Reply:
The thing is, I have no problem accessing the FB_TOKEN environment variable from events.go with the current setup, which is why I'm confused as to why the same doesn't work from constants.go.

Reply:
@abiramen Do you perhaps have `FB_TOKEN` defined in your local environment? `.env` isn't automatically applied (although some editors might). Check your environment variables before `docker-compose up` and see if that might be the issue?

Reply:
Pretty sure the `FB_TOKEN` is not defined in our local environment by design of our system. I will have a crack of this now.

Reply:
@abiramen your facebook token is not being used to get the facebook events. Ive run the production docker files which do not use the facebook token and the events are still being fetched. Have the environment variables even been working from the start. @sergiomercado19 what about the mail api variable? Thoughts?

Reply:
@glebme I've manually tested the MAILJET_TOKEN and it works fine. @abiramen can you send me the other JWT tokens to see what I can do.

Reply:
Yep got those emails.

Reply:
@abiramen disregard my blabbering. I forgot there was dummy data.

Reply:
`.env` files shouldn't be automatically loaded (they're just text files) - although maybe your editor might do it. Either way you can check by opening a terminal in the container and running `printenv` (or similar) on the command line to double check if it's failing at the program level or at the Docker level.

Reply:
Will do this.

Reply:
@abiramen solved your issue.  put "" around your environment for example. "MAILJET_TOKEN=${MAILJET_TOKEN}" or "FB_TOKEN=${FB_TOKEN}" for them to actually be loaded into the container.

@gawdn on a different note. If I can go into the container terminal and do printenv, I'm guessing that's a security issue on your side and theres not much we can do to stop them once they have access to the container?

Reply:
Yep - if they have container access it's a moot point. That is guarded against from my perspective and not something you'd need to worry about.

Reply:
Sounds good.

Reply:
@glebme Having quotes seems like an odd workaround. All the tokens are put in the container's environment; which I manually checked through the container's CLI. 🤔

Reply:
The quotation marks make no difference for me. I can confirm that the variables are being injected in, and the Facebook token is being successfully used from events.go when I enable it:
![image](https://user-images.githubusercontent.com/7523422/94574737-d3769700-02b6-11eb-9b7e-e6d638867aa0.png)

Reply:
I've finally found the issue...

The tests are run from outside the container, and thus can't access the values of environment variables, hence why they fail.